tests/test_model.py

```diff
@@ -280,6 +280,88 @@ def setUp(self):
             rules = harness.model.get_rules("Enabled.Test")
             self.assertEqual(len(rules), 3)
 
+    async def test_mute_full_name(self):
+        """Test mute and unmute by full alarm name.
+        """
+        user = "test_mute_alarm"
+        remote_names = ["ScriptQueue:5", "Test:7"]
+
+        async with EnabledRulesHarness(names=remote_names) as harness:
+
+            harness.model.enable()
+            await harness.model.enable_task
+
+            full_rule_name = f"Enabled.{remote_names[0]}"
+            self.assertIn(full_rule_name, harness.model.rules)
+
+            for rule in harness.model.rules.values():
+                self.assertFalse(rule.alarm.muted)
+                self.assertEqual(rule.alarm.muted_severity, AlarmSeverity.NONE)
+                self.assertEqual(rule.alarm.muted_by, "")
+
+            # mute one rule by full name
+            harness.model.mute_alarm(name=full_rule_name,
+                                     duration=5,
+                                     severity=AlarmSeverity.WARNING,
+                                     user=user)
+            for name, rule in harness.model.rules.items():
```

Comment:
Again, there are two things in the list, so I think unrolling the loop is easier to read and debug.

Reply:
I prefer it as is. If I add more items to the list the test will still pass.